docs: add client file concepts. fixes #66 (#68)
shellscape/webpack-plugin-serve: recipes/multi-entry.md

## 🍲 Setting up Multiple entries

> When building a multi-page application, in order to optimise code/module reuse across the whole application a multi entry configuration is used and developers want to use all the features webpack have, such as `hot module replacement` when one change any file. 

Reviewer: ```suggestion
Certain applications or bundles may require several separate entries. Depending on a user's needs, it may be desirable to make Hot Module Replacement, live-reload, or any other client feature available for each entries, or specific entries other than the default. To accomplish that, the `webpack-plugin-serve/client` file must be included with entries on which client features are needed. We'll show you how that should look below.
```

> In order to take advantage of it the client file has to be present on the dependecy tree to detect and propagate those changes. Sometimes developers get confused because they don't know where exactly to include the client file when facing this scenario.

Reviewer: ```suggestion
```

### Meat and Potatoes

> Given that your configuration is already configured as multi-entry

Reviewer: ```suggestion
Let's say for this example that you have an application which bundles two pages and a web worker separately:
```

```js

> const { WebpackPluginServe: Serve } = require('webpack-plugin-serve');

Reviewer: ```suggestion
```

in the existing config, there probably won't be a definition for this. lets keep it out to avoid confusion

> const serve = new Serve();

Reviewer: ```suggestion
```

module.exports = {
  entry: {
    landing: './landing.js',
    checkout: './checkout.js',
    worker: './worker.js'
  },
  // ...rest of your config
};
```

### Making the Meal

> Now that you have your config, let's include the client file on your entries.

Reviewer: ```suggestion
We'll now add the client script to the entries we'd like to have client-side features added to:
```
